.travis.yml

```diff
@@ -1,15 +1,36 @@
 language: c
 compiler: gcc
-dist: xenial
-before_script:
-- pip install --user redis ramp-packer rmtest
-- git clone -b 4.0.1 --depth 1 https://github.com/antirez/redis.git
-- cd redis
-- make
-- cd ..
+dist: trusty
+
 env:
-- REDIS_MODULE_PATH=$TRAVIS_BUILD_DIR/src/libmodule.so REDIS_PATH=$TRAVIS_BUILD_DIR/redis/src/redis-server
+  - MODULE_NAME=redisgraph
+  - REDIS_MODULE_PATH=$TRAVIS_BUILD_DIR/src/libmodule.so
+  - REDIS_PATH=$TRAVIS_BUILD_DIR/redis/src/redis-server
+
+install:
+  - make clean && make
```

Comment:
I don't see a point in running make before running cmake...

Reply:
agree

Reply:
My bad :)